### PR TITLE
[stable/grafana] allow to define grafana container command in value file

### PR DESCRIPTION
**Pull request by cyrilledec**
Signed-off-by: Cyrille <cyrille@omise.co>

#### What this PR does / why we need it:
This PR allows to define the command to execute when grafana container is starting.
We need to define it in the deployment file if we want our vault third party system be able to wrap the command (https://banzaicloud.com/blog/inject-secrets-into-pods-vault/) and then retrieve secrets from the vault.

#### Special notes for your reviewer:
No change for users who don't want to use this option.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped

### CODE REVIEW

**Review by zanhsieh — APPROVED** [review]

File: stable/grafana/templates/deployment.yaml
Comment (maorfr) [inline stable/grafana/templates/deployment.yaml:133]: `command` seems to be missing in values.yaml

File: stable/grafana/templates/deployment.yaml
Comment (maorfr) [inline stable/grafana/templates/deployment.yaml]: extra space

**Review by maorfr — CHANGES_REQUESTED** [review]

File: stable/grafana/templates/deployment.yaml
Comment (cyrilledec) [reply stable/grafana/templates/deployment.yaml]: fixed

File: stable/grafana/templates/deployment.yaml
Comment (cyrilledec) [reply stable/grafana/templates/deployment.yaml:133]: `command` in value file is optional (this is why it's checked using `if`)

Comment (cyrilledec) [reply stable/grafana/templates/deployment.yaml:133]: we need it only when we want to use HashiCorp Vault (https://banzaicloud.com/blog/inject-secrets-into-pods-vault/)

Comment (maorfr) [reply stable/grafana/templates/deployment.yaml:133]: As a user, I want to know about this.
This should be added to values.yaml and documentd in README.

Comment (cyrilledec) [reply stable/grafana/templates/deployment.yaml:133]: files have been updated

**Comment by maorfr** [comment]
Hi, sorry for the confusion.
What I meant was that we need to have the `command` field in values and in readme for the example, not with the actual required values.

I would leave it commented out even, just so it is visible for users.

And if args is the only field there, can we do without it?

Sorry for the misunderstanding. Thanks again!

**Comment by cyrilledec** [comment]
Hi, no problem I've updated values file and also removed `args` as it's the only field. README also updated

File: stable/grafana/README.md
Comment (maorfr) [inline stable/grafana/README.md]: You should probably update the default value?

Comment (cyrilledec) [reply stable/grafana/README.md]: updated

File: stable/grafana/README.md
Comment (maorfr) [inline stable/grafana/README.md]: The defailt value should be `nil`, as the `command` field is commented, similarly to `priorityClassName` for example. After this we can merge! :)

Comment (cyrilledec) [reply stable/grafana/README.md]: updated

**Comment by maorfr** [comment]
/hold cancel
/lgtm

Thanks a lot for this contribution!

**Pull request merged**
